box-tests.sh

```diff
@@ -28,11 +21,17 @@ runtests35-sqlite3 auth_tests
 runtests35-postgres auth_tests
 runtests35-mysql auth_tests
 
-runtests27-sqlite3 admin_widgets --selenium chrome
+runtests36-sqlite3-gis gis_tests
+runtests36-postgres-gis gis_tests
+runtests36-mysql-gis gis_tests
+runtests36-sqlite3 auth_tests
+runtests36-postgres auth_tests
+runtests36-mysql auth_tests
+
+runtests36-sqlite3 admin_widgets --selenium chrome --parallel 1
```

Comment:
I don't want to execute every selenium test - just a few of them to validate that selenium testing works in general.